addons/calendar/models/calendar_event.py

```diff
@@ -618,6 +618,14 @@ def _attendees_values(self, partner_commands):
 
         removed_partner_ids = []
         added_partner_ids = []
+
+        # if commands are just integers, assume they are ids with the intent to `Command.set`
+        if partner_commands and isinstance(partner_commands[0], int):
```

Comment:
Can't we simplify to:

`partner_commands = [(4, partner_id) for partner_id in partner_commands]`

And let the rest of the code do its job normally?

Reply:
It's just that we'll loop over stuff for no reason. But the diff is indeed smaller

Reply:
Ah yes, the "6", rarely ever use that one so I forgot it exists :smile: 

That way we have a simple conversion before the actual logic, seems safer if we ever change the below logic (unlikely but well).